tokenizer/src/commonMain/kotlin/com/xebia/functional/tokenizer/ModelType.kt

```diff
@@ -84,6 +84,9 @@ sealed class ModelType(
   object CODE_SEARCH_BABBAGE_CODE_001 : ModelType("code-search-babbage-code-001", R50K_BASE, 2046)
   object CODE_SEARCH_ADA_CODE_001 : ModelType("code-search-ada-code-001", R50K_BASE, 2046)
 
+  class FineTunedModel(name: String, val baseModel: ModelType)
```

Comment:
Could you format the code?

Reply:
I ran spotless on it... Isn't this enough?